clients/web/src/lib/api/queries/RestrictedPlayers.store.ts

```diff
@@ -20,32 +20,30 @@ interface Member {
     profile: MemberProfile;
 }
 
-export interface MemberRestrictionEvent {
+export interface MemberRestriction {
     id: number;
 
-    eventType: number;
-
-    message?: string;
+    type: MemberRestrictionType;
 
-    restriction: {
-        type: MemberRestrictionType;
+    expiration: Date;
 
-        expiration: Date;
+    reason: string;
 
-        reason: string;
+    member: Member;
 
-        member: Member;
+    manualExpiration?: Date;
 
-        manualExpiration?: Date;
+    manualExpirationReason?: string;
 
-        manualExpirationReason?: string;
+    memberId: number;
+}
 
-        memberId: number;
-    };
+export interface MemberRestrictionEvent extends MemberRestriction{
+    eventType: number;
```

Comment:
Should this be a number or enum for clarity?

Reply:
Actually, it's used as an enum in the switch case below?

Reply:
done